src/main.rs

```diff
@@ -83,15 +81,16 @@ pub struct Options {
 fn main() {
     cli_batteries::run(
         version!(crypto, small_powers_of_tau),
-        async_main::<TestTranscript>,
+        async_main::<kzg_ceremony_crypto::contribution::Transcript>,
     );
 }
 
 async fn async_main<T>(options: Options) -> EyreResult<()>
 where
-    T: Transcript + Send + Sync + 'static,
+    T: kzg_ceremony_crypto::interface::Transcript + Send + Sync + 'static,
```

Comment:
Same here. We can use and `as` clause or keep the `interface::` part to disambiguate the trait and instance.